hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/io/storage/row/HoodieRowCreateHandle.java

```diff
@@ -241,6 +242,9 @@ public WriteStatus close() throws IOException {
     stat.setTotalWriteBytes(fileSizeInBytes);
     stat.setFileSizeInBytes(fileSizeInBytes);
     stat.setTotalWriteErrors(writeStatus.getTotalErrorRecords());
+    for (Pair<HoodieRecordDelegate, Throwable> pair : writeStatus.getFailedRecords()) {
+      LOG.info("Failed to write {}", pair.getLeft(), pair.getRight());
```

Comment:
Agreed we should use ERROR log level, but using two `{}` doesn't make sense to me.

It will run `Throwable.toString()` and then print only the short description. Stacktrace won't be printed. Instead we should directly pass the Throwable to the logging API.